js/util/navigationUtil.ts

```diff
@@ -25,6 +25,10 @@ export default class NavigationUtil {
     return null;
   }
 
+  static isEnabled(): boolean {
```

Comment:
Is "enabled" actually what we're tracking here?  I thought it was whether we were using an HOC.  Are they two ways to say the same thing?

Reply:
This is effectively "has the dev enabled screenview autocapture via the HOC?"

Maybe `isHocEnabled()` would be better?